app/ethWallet-geth.js

```diff
@@ -185,9 +185,12 @@ const cleanupGethAndExit = (exitCode) => {
 }
 
 const cleanupGeth = (processId) => {
-  processId = processId || gethProcessId
+  processId = (
+    (processId || gethProcessId) ||
```

Comment:
is the extra pair of parens meaningful? Seems like it wouldn't change anything to remove them and the line would be read the same:

> get processId or if it doesn't exist gethProcessId or if geth was running get geth pid

Reply:
Eh, sometimes I go a little extreme on the parens :) I can clean it up